### PR TITLE
fix(whats-new): swapping update types and reduce vertical space

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
Screen Shot 2021-07-21 at 11 24 19 AM

**After:**
Screen Shot 2021-07-21 at 11 23 53 AM

### CODE REVIEW

**Review by dcramer — APPROVED** [review]

File: static/app/components/sidebar/broadcastSdkUpdates.tsx [hunk 1]
```diff
@@ -100,7 +100,6 @@ const UpdatesList = styled('div')`
 `;
 
 const Suggestions = styled('div')`
-  margin-top: ${space(1)};
```
Comment (EvanPurkhiser) [inline static/app/components/sidebar/broadcastSdkUpdates.tsx:103]: Can you show what this looks like with multiple notifications? I’m not sure this is safe to remove?

Comment (doralchan) [reply static/app/components/sidebar/broadcastSdkUpdates.tsx:103]: Yeah there's still spacing between the updates if there are multiple under each project (which from my understanding is less frequent) 

Screen Shot 2021-07-21 at 12 25 03 PM

Comment (EvanPurkhiser) [reply static/app/components/sidebar/broadcastSdkUpdates.tsx:103]: Sorry I mean when there’s multiple projects. I’m just trying to understand why these margins needed removed

Comment (doralchan) [reply static/app/components/sidebar/broadcastSdkUpdates.tsx:103]: Ah each project has a margin already so this was acting more like line-height than anything else. Should be fine to remove

**Pull request merged**